### PR TITLE
add cofix space

### PR DESCRIPTION
**Pull request by AndyChenW**
**Add space for ...**  
  
Etherscan: https://etherscan.io/address/0x1a23a6BfBAdB59fa563008c0fB7cf96dfCF34Ea1  
CoinGecko: https://www.coingecko.com/en/coins/cofix
Website: https://cofix.io 
Twitter: https://twitter.com/cofixprotocol

### CODE REVIEW

File: spaces/cofix/index.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "CoFiX",
+  "network": "1",
+  "symbol": "COFI",
+  "skin": "cofi",
```
Comment (gokaiorg) [inline spaces/cofix/index.json:5]: Skin field value should be similar to file name.
```suggestion
  "skin": "cofix",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Please use square format for your images.

File: spaces/cofix/index.json (hunk 1 above)
Comment (AndyChenW) [reply spaces/cofix/index.json]: fixed.

**Comment by AndyChenW** [comment]
> Please use square format for your images.

make the logo same as the space.png now :)

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**